<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JSON/XML Parser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1" colspan="2">
     <widget class="QLineEdit" name="editValue"/>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="editKey"/>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QPushButton" name="buttonDumpToFile">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Dump to file</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="buttonGet">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Get</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QPushButton" name="buttonSelectFile">
      <property name="text">
       <string>Select File</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="buttonSet">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelKey">
      <property name="text">
       <string>Key:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelValue">
      <property name="text">
       <string>Value:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QLabel" name="labelMessage">
      <property name="text">
       <string>No file loaded.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>343</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
